src/java.desktop/share/classes/javax/swing/plaf/basic/BasicTextUI.java

```diff
@@ -1029,10 +1029,9 @@ public Dimension getMaximumSize(JComponent c) {
 
 
     /**
-     * Gets the allocation to give the root View.  Due
-     * to an unfortunate set of historical events this
-     * method is inappropriately named.  The Rectangle
-     * returned has nothing to do with visibility.
+     * Gets the allocation to give the root View.
+     * The Rectangle returned has nothing to do with visibility but
```

Comment:
```suggestion
     * The rectangle returned has nothing to do with visibility but
```
I suggest going lower case unless you specifically refer to the rectangle as the type in which case, it should be `{@code Rectangle}`.